sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/StoreResult.java

```diff
@@ -208,6 +208,9 @@ public void serialize(StoreResult storeResult,
             jsonGenerator.writeObjectField("transportRequestTimeline", storeResult.storeResponse != null ?
                 storeResult.storeResponse.getRequestTimeline() :
                 storeResult.exception != null ? BridgeInternal.getRequestTimeline(storeResult.exception) : null);
+            jsonGenerator.writeObjectField("transportChannelAcquisitionContext", storeResult.storeResponse != null ?
```

Comment:
Let's call it `transportRequestChannelAcquisitionContext` because it is per request.

Reply:
updated